### PR TITLE
Add relation between customer and idea models

### PR DESCRIPTION
**Pull request by MiBBy-4**

### CODE REVIEW

File: app/controllers/concerns/current_customer_concern.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-  include do
+  included do
     before_action :set_current_customer
   end
 
   def set_current_customer
     if session[:customer_id]
-      @current_customer = Customer.find(session[:customer_id])
+      @current_customer = Customer.find_by_id(session[:customer_id])
```
Comment (AndreyAnischik) [inline app/controllers/concerns/current_customer_concern.rb:10]: why did you change find method?

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -7,11 +7,20 @@
...
 
     resource '*',
       headers: :any,
       methods: [:get, :post, :put, :patch, :delete, :options, :head],
-      expose: %w[Authorization Uid]    
+      credentials: true
+  end
+
+  allow do
+    origins 'https://mib-idea-api.herokuapp.com'
```
Comment (AndreyAnischik) [inline config/initializers/cors.rb:19]: use env variable

**Review by AndreyAnischik — CHANGES_REQUESTED** [review]

File: app/controllers/concerns/current_customer_concern.rb (hunk 1 above)
Comment (MiBBy-4) [reply app/controllers/concerns/current_customer_concern.rb:10]: I don't remember why I did it. I will bring it back

**Review by AndreyAnischik — APPROVED** [review]
